twitter-langchain/pyproject.toml

```diff
@@ -11,7 +11,8 @@ python = "^3.10"
 pydantic = "^2.9.2"
 langchain = "^0.3.7"
 tweepy = "^4.14.0"
-
+cdp-agentkit-core = "^0.0.1"
+cdp-langchain = "^0.0.1"
```

Comment:
where does the `cdp-langchain` dep come from? This package should only depend on `cdp-agentkit-core`